test/e2e/gardener/common.go

```diff
@@ -113,12 +114,20 @@ func DefaultShoot(name string) *gardencorev1beta1.Shoot {
 				{
 					Type: "local-ext-shoot",
 				},
-				{
-					Type: "local-ext-shoot-after-worker",
-				},
 			},
 		},
 	}
+
+	if ginkgo.Label("default").MatchesLabelFilter(ginkgo.GinkgoLabelFilter()) {
+		// TODO(maboehm) The extension is not available in the previous gardener
```

Comment:
Please state when this code can be removed.
See https://github.com/gardener/gardener/blob/master/docs/development/process.md#todo-statements